pkg/lexer/lexer_test.go

```diff
@@ -0,0 +1,241 @@
+package lexer_test
+
+import (
+	"testing"
+
+	"github.com/raklaptudirm/mash/pkg/lexer"
+	"github.com/raklaptudirm/mash/pkg/token"
+)
+
+func TestLexer(t *testing.T) {
+	input := `# comment line
+for
+if
+elif
+else
+
+let
+func
+
+break
+continue
+return
+
+let identifier
+let 3141592653
+let "a string"
+
+# line comment
+let i # inline
+
+let +
+let -
+let *
+let /
+let %
+
+let &
+let |
+let ^
+let <<
+let >>
+let &^
+
+let +=
+let -=
+let *=
+let /=
+let %=
+
+let &=
+let |=
+let ^=
+let <<=
+let >>=
+let &^=
+
+let &&
+let ||
+
+let ==
+let <
+let >
+let =
+let :=
+let !
+
+let !=
+let <=
+let >=
+
+let (
+let [
+let {
+let ,
+
+let )
+let ]
+let }
+let ;
+let :
+
+break
+
+echo a command
+||&&!|
+`
```

Comment:
Is there a specific reason for removing the `echo a+b>>README` case?

Reply:
I have decided to remove the `<`, `>`, and `>>` operators from commands. I did this after a thorough consideration, and here are my reasons:
- The operators add a lot of unnecessary edge cases:
  - `cat main.go | grep TODO < main.go`
  - `echo Hello World > a.txt >> b.txt`
- The function of those operators can be easily achieved by using simple commands:
  - `command < file` =  `cat file | command`
  - `command > file` = ` command | w file` (`w` will be an inbuilt command)
  - `command >> file` = `command | a file` (`a` will be an inbuilt command)

In conclusion, the number of problems that adding these as operators will add can be easily solved by adding some inbuilt commands. The disadvantages are more numerous than any advantages. Let me know what you think.